test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Image.cshtml

```diff
@@ -13,10 +13,19 @@
     <!-- Plain image tag -->
     <img src="~/images/red.png" alt="Red block" title="&lt;the title>">
 
-    <!-- Plain image  tag with file version -->
+    <!-- Plain image tag with file version -->
     <img src="~/images/red.png" alt="Red versioned" title="Red versioned" asp-append-version="true" />
 
-    <!-- Plain image  tag with file version set to false -->
+    <!-- Plain image tag with file version set to false -->
     <img src="~/images/red.png" alt="Red explicitly not versioned" title="Red versioned" asp-append-version="false" />
+
+    <!-- Plain image tag with absolute path and file version -->
+    <img src="http://contoso.com/hello/world" alt="Absolute path versioned" asp-append-version="true" />
+
+    <!-- Plain image tag with file version and path to file that does not exist -->
+    <img src="~/images/fake.png" alt="Path to non existing file" asp-append-version="true" />
+
+    <!-- Plain image tag with file version and path containing query string -->
+    <img src="~/images/red.png?abc=def" alt="Path with query string" asp-append-version="true" />
```

Comment:
It will just return it without adding file version. I've added that case to the test.